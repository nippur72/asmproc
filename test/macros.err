   mac M1_M
   ; THIS IS MACRO MEM
   endm

   mac M1_C
   ; THIS IS MACRO CONST
   endm

   mac M1_I
   ; THIS IS MACRO INDIRECT
   endm

   mac M1__A
   ; THIS IS MACRO STRING
   endm

MACRO M1 "A"
   ; THIS IS MACRO STRING
END MACRO

MACRO M1 "A", MEM, CONST, INDIRECT
   ; THIS IS MACRO STRING
END MACRO

TEXT
M1 $AAAA
M1 #$AAAA
M1 ($AAAA)
M1 "A"
M1 "A", 3, #12, ($22)
TEXT

