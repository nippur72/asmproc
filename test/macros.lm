macro M1 mem
   ; THIS IS MACRO mem
end macro

macro M1 const
   ; THIS IS MACRO const
end macro

macro M1 indirect
   ; THIS IS MACRO indirect
end macro

macro M1 "a"
   ; THIS IS MACRO string
end macro

macro M1 "a", mem, const, indirect
   ; THIS IS MACRO string
end macro

TEXT
M1 $aaaa
M1 #$aaaa
M1 ($aaaa)
M1 "a"
M1 "a", 3, #12, ($22)
TEXT

