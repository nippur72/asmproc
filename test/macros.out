   mac M1_M
   ; THIS IS MACRO MEM
   endm

   mac M1_C
   ; THIS IS MACRO CONST
   endm

   mac M1_I
   ; THIS IS MACRO INDIRECT
   endm

   mac M1__A
   ; THIS IS MACRO STRING
   endm

   mac M1__A_M_C_I
   ; THIS IS MACRO STRING
   endm

TEXT
   M1_M $AAAA 
   M1_C #$AAAA 
   M1_I ($AAAA) 
   M1__A "A" 
   M1__A_M_C_I "A", 3, #12, ($22) 
TEXT

